py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -82,6 +82,10 @@
 TEKTON_CLUSTER_NAME = "kf-ci-v1"
 TEKTON_CLUSTER_ZONE = "us-east1-d"
 
+if sys.version_info < (3,0):
```

Comment:
Would it be better to use "if six.PY2"

Reply:
Sure never used, `six` before actually :) . Are Kubeflow currently using `six` or are we adding a new dependency?

Reply:
Found that it seems to be used within the current code base so will update and resolve.